pkg/cli/setup/application.go

```diff
@@ -47,6 +50,18 @@ resource demo 'Applications.Core/containers@2023-10-01-preview' = {
 	radYamlTemplate = `workspace:
   application: %q
 ` // Trailing newline intentional.
+
+	bicepConfigTemplate = `{
+	"experimentalFeaturesEnabled": {
+		"extensibility": true,
+		"providerRegistry": true,
+		"dynamicTypeLoading": true
+	},
+	"providers": {
+		"radius": "br:biceptypes.azurecr.io/radius:%s",
```

Comment:
Originally had this set to "latest" but I updated this to correspond with the current version. So "latest" will only be used on edge versions and the current release version is used otherwise

Reply:
Good plan 👍